MetaMask/metamask-extension
# Add links to release headers

Each release header now includes a link to the range of commits included with that release. These links are at the end of the document, in accordance with the ["keep a changelog"](https://keepachangelog.com/en/1.0.0/) format.

For the purpose of this changelog, the "previous release" is the most recent release mentioned in the changelogs. The diffs ignore any releases that were omitted from the changelog. This is mainly an issue with older releases, so it seemed acceptable. All releases have been documented for a couple of years now, and will be going forward as well.

The name of the "Current Develop Branch" section was changed to "Unreleased" to confirm with "keep a changelog".

The `auto-changelog.js` script has been updated to update/add these links whenever adding a new release header as well.

Relates to #10752

Manual testing steps:  

For development changelog updates:
- Run `yarn update-changelog`
- See that it places changelog entries under 'Unreleased`

For RC changelog updates with existing release header:
- Bump the `version` in `app/manifest/_base.json`, to simulate an RC-like environment
- Add a new release header to the changelog for the new version
- Run `yarn update-changelog`
- See that it places changelog entries under the new release header along with a new release header link, and it updates the `Unreleased` link.
 
For updates without a release header:
- Bump the `version` in `app/manifest/_base.json`, to simulate an RC-like environment
- Run `yarn update-changelog`
- See that it places changelog entries under the newly added release header
